contracts/cosmwasm-vm/cw-ibc-core/tests/channel/test_packet.rs

```diff
@@ -105,7 +105,16 @@ fn test_packet_send() {
         )
         .unwrap();
 
```

Comment:
Can we have tests for where authencticate capability is used?

Reply:
added test for it . Also there will be a separate task to address test coverage as well for critical portions.